### ISSUE TITLE
build: Misleading prompt during local build

### ISSUE THREAD

**Issue by arimer** [IETS3/iets3.opensource#905]
When building maintenance/mps20222 locally with a configured JDK 17 the following missleading information pops up.

```
> Configure project :
Local build detected, version will be 2022.2-SNAPSHOT
MPS requires Java 11 but current JVM uses 17, starting MPS will most probably fail!
```

This info is on wrong in multiple ways
- JDK 17 should be fine for building projects depend on >= MPS 2022.2
- iets3.os build is managing and downloading a compatible JBR version which is used during the build. So Only a java version for running the build.gradle file would be required

**Comment by alexanderpann** [comment]
I have no clue where this message is supposed to be located and I don't see it locally or on the CI.

**Comment by arimer** [comment]
How to reproduce:
1. Checkout maintenance/mps20222 brach
2. Use a local JDK installation in a version > 11
3. Run the `./gradlew`

Example of a local build
<img width="672" alt="grafik" src="https://github.com/IETS3/iets3.opensource/assets/16648247/6b83179f-f1c9-4703-b3eb-e7c530aad966">

**Comment by arimer** [comment]
> I don't see it locally or on the CI.
Most probably our TC agens are still using JDK 11 as well as your local setup.

**Comment by alexanderpann** [comment]
Yes, it comes from the mps-gradle-plugin: [https://github.com/mbeddr/mps-gradle-plugin/blob/a54f29f24d3155ecacd21d64de2b7a0f07ce660e/src/main/kotlin/de/itemis/mps/gradle/Common.kt#L69](https://github.com/mbeddr/mps-gradle-plugin/blob/a54f29f24d3155ecacd21d64de2b7a0f07ce660e/src/main/kotlin/de/itemis/mps/gradle/Common.kt#L69)

**Comment by arimer** [comment]
fixed with #906
